ledger-tool/src/main.rs

```diff
@@ -2021,13 +2028,19 @@ fn main() {
                     accounts_index_config.drives = Some(accounts_index_paths);
                 }
 
-                let filler_account_count =
-                    value_t!(arg_matches, "accounts_filler_count", usize).ok();
+                let mut filler_accounts_config = FillerAccountsConfig::default();
+                if let Some(count) = value_t!(arg_matches, "accounts_filler_count", usize).ok() {
+                    filler_accounts_config.count = Some(count);
+                }
+
+                if let Some(size) = value_t!(arg_matches, "accounts_filler_size", usize).ok() {
+                    filler_accounts_config.size = Some(size);
+                }
```

Comment:
Can probably set the Clap args to [have a default value](https://docs.rs/clap/latest/clap/struct.Arg.html#method.default_value) of `0`, and then something like:

```rust
let filler_accounts_config = FillerAccountsConfig {
    count: value_t!(...),
    size: value_t!(...),
};
```